lms/djangoapps/courseware/views.py

```diff
@@ -423,6 +437,11 @@ def _index_bulk_op(request, course_key, chapter, section, position):
 
         context['show_chat'] = show_chat
 
+        if settings.FEATURES.get('ENTRANCE_EXAMS', False) and getattr(course, 'entrance_exam_enabled', False):
+            _, exam_current_score = get_entrance_exam_content_info(request, course_module, course)
```

Comment:
I think the convention here is to use double underscores as the single underscore is reserved for I18N, e.g. _('...')